### DISCUSSION TITLE
How to fix these warnings?

### DISCUSSION THREAD

**Discussion by rahaaatul** [mar10/wsgidav#323] [Q&A]
```shell
WARNING : App wsgidav.mw.cors.Cors(None).is_disabled() returned True: skipping.
```

```shell
WARNING : Share '/:dir_browser' will allow anonymous write access.
```

```shell
16:38:04.787 - INFO    : 192.168.1.100 - RGX - [2024-08-02 10:38:04] "HEAD /King Arthur Legend of the Sword (2017)/" elap=0.002sec -> 200 OK
16:38:04.813 - INFO    : 192.168.1.100 - (anonymous) - [2024-08-02 10:38:04] "PROPFIND /King Arthur Legend of the Sword (2017)/" depth=1, range=bytes=0-, elap=0.001sec -> 401 Not Authorized
```

**Comment by blitzjunge** [comment]
Same question here. I did a little research, dived into the source code but could not find an answer. 
In order to improve security I just disabled the dir_browser in the config file.
With this I am not able to browse the directory structure via a web browser. But I can still use all my apps without problems and can also browse the directory structure with Cyberduck, macOS Finder, etc.

**Comment by mar10** [comment]
> WARNING : App wsgidav.mw.cors.Cors(None).is_disabled() returned True: skipping.

This warning can be removed by enabling the cors plugin

> WARNING : Share '/:dir_browser' will allow anonymous write access.

This message is issued, for every share that allows anonymous access. However `:dir_browser` is a built-in share that only contains the static assets. It is not used to serve any file content.

**Reply by blitzjunge** [reply]
Thank you for the clarification.
